src/database/DatabaseAccessor.java

```diff
@@ -980,12 +980,16 @@ public void addGeneratedProfiles(String expId, String folderPath,
             if (!f.getName().equals(inputFileName)) {
 
                 String checkSumMD5;
+                long fileSize;
                 try (FileInputStream is = new FileInputStream(f)) {
                     checkSumMD5 = DigestUtils.md5Hex(is);
                 }
+
+                fileSize = f.length();
+
                 fileMethods.addGeneratedFile(e.getID(), FileTuple.PROFILE,
                         f.getPath(), inputFileName, metaData, uploader,
-                        isPrivate, grVersion, checkSumMD5);
+                        isPrivate, grVersion, checkSumMD5, fileSize);
             }
         }
     }
```

Comment:
Just do `fileMethods.addGeneratedFile(e.getID(), FileTuple.PROFILE,
                        f.getPath(), inputFileName, metaData, uploader,
                        isPrivate, grVersion, checkSumMD5, f.length());` without the unnecessary declaration and initialization.